tools/python3/cmake.py

```diff
@@ -0,0 +1,48 @@
+from tools.python3 import decode_bytes, run
```

Comment:
this file doesn't exist on my branch anymore

https://github.com/PhilipDeegan/PHARE/tree/3d_base/tools/python3

Reply:
ok good to know, I think I will use meld to see the diff more easily.